### ISSUE TITLE
Failed to execute 'importScripts' on 'WorkerGlobalScope'

### ISSUE THREAD

**Issue by jonaro00** [CoderLine/alphaTab#1400]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

On version 1.3.0-alpha.938 and later, alphaTab fails to render and this error appears twice in the Console.
`Uncaught DOMException: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'https://cdn.jsdelivr.net/npm/@coderline/alphatab@1.3.0-alpha.938/dist/alphaTab.js' failed to load.
    at blob:http://127.0.0.1:8000/f7d3e53e-53e3-47de-8150-6daacba5fcfe:1:1`

### Expected Behavior

On version 1.3.0-alpha.933, it works like it should.

### Steps To Reproduce

This happens in the project https://github.com/jonaro00/TabPortal (plain HTML+CSS+JS with alphaTab from CDN)
Not sure if there is anything with my use-case triggering this.

### Link to jsFiddle, CodePen, Project

https://github.com/jonaro00/TabPortal

### Found in Version

1.3-alpha

### Platform

Web

### Environment

```markdown
- **OS**: Ubuntu
- **Browser**: Brave Version 1.63.165 Chromium: 122.0.6261.94 (Official Build) (64-bit)
```

### Anything else?

Based on the timing, this was likely caused by https://github.com/CoderLine/alphaTab/pull/1393 or https://github.com/CoderLine/alphaTab/pull/1386

**Comment by Danielku15** [comment]
Can you please provide a minimal repoducible example which doesn't require setting up a whole development environment for rust?

**Comment by jonaro00** [comment]
Sure, I quickly pasted all parts into one standalone file:
https://gist.github.com/jonaro00/e623e47e29f285373dd496ff67e46c66
I can strip away more things if this one is too hard to debug.

**Comment by Danielku15** [comment]
Thanks for providing the standalone snippet. Could reproduce the problem and its fixed in #1401
